installer/conf/omsagent.d/npmd.conf

```diff
@@ -1,9 +1,9 @@
 <source>
     type npmd
     tag oms.npmd
-    location_unix_endpoint  "/var/opt/microsoft/omsagent/run/npmdagent.sock"
-    location_control_data   "/etc/opt/microsoft/omsagent/conf/npmd_agent_config.xml"
+    omsadmin_conf_path      "%CONF_WS_DIR%/omsadmin.conf"
```

Comment:
In addition, how are you replacing this string? Will your DSC module copy down this conf file and replace the string then?